src/Cards/CurrentCard.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  Card,
+  Question,
+  Option,
+  SubmitButton,
+  CardWindowBar,
+  ButtonContainer,
+} from "./styled";
+
+import close from "../images/close.svg";
+
+const CurrentCard = ({
+  question,
+  setScore,
+  score,
+  getNextQuestion,
+  isSubmit,
+  setIsSubmit,
+  isCorrect,
+  setIsCorrect,
+  isIncorrect,
+  setIsIncorrect,
+  questionNumber,
+}) => {
+  const [selectedOption, setSelectedOption] = useState("");
+  const [isEmpty, setIsEmpty] = useState(false);
+
+  function setSelectedValue(option) {
+    if (!isSubmit) {
+      setSelectedOption(option);
+    }
+  }
+  function checkSelect(selected) {
```

Comment:
A better name for this would be `checkIfSelected`. 
Bonus points: you can also just return the equality check since it will return true or false. 
`const checkSelected = (selected) => selected === selectedOption`